src/main/java/io/jenkins/plugins/audit/helpers/Factory.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.plugins.audit.helpers;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.TimeZone;
+
+public class Factory {
+    /*
+     * Returns a date string in the ISO-8601 format
+     */
+    public static String dateTimeFormat() {
+
+    	TimeZone tz = Calendar.getInstance().getTimeZone();
+        ZonedDateTime d = ZonedDateTime.of(LocalDateTime.now(), ZoneId.of(tz.getID()));
```

Comment:
Looks like you can just use `ZonedDateTime.now()` to do the same. Thus, you can simplify all this down to:

```
return ZonedDateTime.now().format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
```

However, I'd suggest renaming the method to something like `currentDateTimeFormatted()`, or add a `ZonedDateTime` or similar parameter.